libs/localization/src/lib/en.json

```diff
@@ -262,10 +262,15 @@
       "save": "Save"
     },
     "DropdownHeaders": {
+      "errorMetric": "Enable error bars",
       "fairnessMetric": "Fairness metric",
       "performanceMetric": "Performance metric",
       "sensitiveFeature": "Sensitive feature"
     },
+    "ErrorBounds": {
+      "howToRead": "How to read these error bars",
+      "introModalText": "Error bars show the uncertainty in the metric values due to the size of the evaluation data set. We show 95% confidence intervals calculated using Wilson scores (for binary classification) or normal approximation (for other tasks)."
```

Comment:
Is this the final version agreed with the PMs? Weren't we going to put in some caveat about 'These are only approximate, and you data may violate the assumptions made' ?

Reply:
And I've just seen the email showing the current text came from Miro. I'll resolve this.